projects/plugins/jetpack/modules/likes/jetpack-likes-master-iframe.php

```diff
@@ -34,6 +34,17 @@ function jetpack_likes_master_iframe() {
 		$new_layout
 	);
 
+	/**
+	 * Filters the Likes iframe src, if any parameters need to be overridden or tracked.
+	 *
+	 * @module likes
+	 *
+	 * @since X.X.X
```

Comment:
Thanks for adding the docblock!

You can use `$$next-version$$`, it will be automatically replaced by the correct version number on release:

```suggestion
	 * @since $$next-version$$
```

For more info:
https://github.com/Automattic/jetpack/blob/d3f58235992e84fcc8a4b1bc191c56ba6edf213a/projects/packages/README.md#L116-L123